apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/plugins/editor-template-classes/index.js

```diff
@@ -23,21 +24,26 @@ const EditorTemplateClasses = withSelect( ( select ) => {
 	} );
 	return { templateClasses };
 } )( ( { templateClasses } ) => {
-	const blockListInception = setInterval( () => {
-		const blockListParent = document.querySelector( '.block-editor-writing-flow' );
+	useEffect( () => {
+		const blockListInception = setInterval( () => {
+			const blockListParent = document.querySelector(
+				'.editor-styles-wrapper > .block-editor-block-list__layout'
+			);
 
-		if ( ! blockListParent ) {
-			return;
-		}
-		clearInterval( blockListInception );
+			if ( ! blockListParent ) {
+				return;
+			}
+			clearInterval( blockListInception );
 
-		blockListParent.className = classNames(
-			'block-editor-writing-flow',
-			'a8c-template-editor fse-template-part',
-			...templateClasses
-		);
-	} );
+			blockListParent.className = classNames(
+				blockListParent.className,
+				'a8c-template-editor fse-template-part',
+				...templateClasses
+			);
+		} );
 
+		return () => clearInterval( blockListInception );
```

Comment:
By the time the component unmounts, will there be a reference to `blockListInception` to clear if it's already cleared on L36?

Reply:
I would think so.  `blockListInception` shouldn't be removed from memory when `clearInterval` is called with it, but when the variable is no longer referenced in code.  Since this useEffect block is still defined until the component unmount, that reference should still be there when this return is called.